config/environments/production.rb

```diff
@@ -19,8 +19,7 @@
   config.assets.compress = true
 
   # Fallback to the asset pipeline if a precompiled asset is missed.
```

Comment:
Is this comment incorrect now?  We're either going to serve a precompiled asset or it's going to raise an error, right?

Reply:
Hm, should I just drop it? It's a default Rails comment :wink:

Reply:
I have a feeling we changed it... the default is now:

```
  # Do not fallback to assets pipeline if a precompiled asset is missed.
  config.assets.compile = false
```

We're sneaky...

from [here](https://github.com/rails/rails/blob/ec387c6dd975fecea69db2ed9ff0c090ac59cf83/railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt#L34-L35)

Reply:
🙏 git blame doesn't say I did it